.env.dist

```diff
@@ -3,6 +3,8 @@
 APP_DIR=/home/zola/zola-app
 NODE_ENV=development
 NO_DEBUG=false
+API_HOST=localhost
```

Comment:
This is used to compute an `API_HREF` in `@constants/api`. It is then used in the e-mail (the link opens to the route).